app/controllers/users/registrations_controller.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+class Users::RegistrationsController < Devise::RegistrationsController
+  before_action :configure_sign_up_params, only: [:create]
```

Comment:
According to Ragnarson's Ruby style guide, I would change `[:create]` to `%i(create)`.